### DISCUSSION TITLE
First pass at this repo

### DISCUSSION THREAD

**Discussion by kowh-ai** [ckan/ckan-docker-base#2] [General]
@amercader - this is the first pass at this repo

- all images build and push ok
- they have been (basic) tested using https://github.com/ckan/ckan-docker
- I'm wondering if I need to setup a GitHub action to build the images also
- Scanning the images for vulnerabilities can come later through a PR

**Comment by amercader** [comment]
:tada: that is great! I can have a look next week

I found this project which seems like we could leverage to create some basic tests for the images (not now, we can create a separate issue for that): 

https://github.com/NVIDIA/container-canary

**Reply by kowh-ai** [reply]
Included some setup instructions and a yaml file with 3 basic tests to start with...
